app/controllers/manifestation_controller.rb

```diff
@@ -840,45 +836,31 @@ def refuse_unreasonable_page
 
   def prep_for_print
     @m = Manifestation.find(params[:id])
-    if @m.nil?
-      head :ok
-    else
-      unless @m.published?
-        flash[:notice] = t(:work_not_available)
-        redirect_to '/'
-        return
-      end
 
-      @e = @m.expression
-      @w = @e.work
-      @author = @w.authors[0] # TODO: handle multiple authors
-      unless is_spider?
-        Chewy.strategy(:bypass) do
-          @m.record_timestamps = false # avoid the impression count touching the datestamp
-          impressionist(@m)
-          @m.update_impression
-          unless @author.nil?
-            @author.record_timestamps = false # avoid the impression count touching the datestamp
-            impressionist(@author) # also increment the author's popularity counter
-            @author.update_impression
-          end
-        end
-        # log the read, for later recommendation feature using the Disco gem
-        ahoy.track 'text read or printed', text_id: @m.id, title: @m.title, author: @m.author_string
-      end
-      if @author.nil?
-        @author = Authority.new(name: '?')
-      end
-      @translators = @m.translators
-      @illustrators = @m.involved_authorities_by_role(:illustrator)
-      @editors = @m.involved_authorities_by_role(:editor)
-      @page_title = "#{@m.title_and_authors} - #{t(:default_page_title)}"
-      if @print
-        # remove MMD's automatic figcaptions
-        @html = MultiMarkdown.new(@m.markdown).to_html
-                             .force_encoding('UTF-8').gsub(%r{<figcaption>.*?</figcaption>}, '')
-      end
+    unless @m.published?
+      flash[:notice] = t(:work_not_available)
+      redirect_to '/'
+      return
     end
+
+    @e = @m.expression
+    @w = @e.work
+    @author = @w.authors[0] # TODO: handle multiple authors
+    track_view(@m)
+    track_view(@author)
+
+    if @author.nil?
+      @author = Authority.new(name: '?')
+    end
+    @translators = @m.translators
```

Comment:
I would also like to track a view for every involved translator. It's an oversight that we weren't.
(but no need for editors, illustrators, and other roles.)